package.json

```diff
@@ -1,6 +1,11 @@
 {
   "name": "wonka-ui",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
Do we have to match the package.json for all subdirectories?

Reply:
Not necessarily. The demo-app can have a different version number than the wonka-ui library. Also, this package.json (at the root) does not need the name, version, license and repository fields. I can try to clean that.